src/main/java/pl/orki/hackathon/webapp/user/entity/User.java

```diff
@@ -26,10 +25,12 @@
     @Column(name = "role")
     private Role role;
 
+    @Column(name = "role_id")
+    private Long roleId;
```

Comment:
If we want to store Role as a separate entity on the database (however I am not sure if it's necessary and enum may be enough for now), then in Hibernate we need to create one to one mapping with `Role` object, not it's id - joining based on id will be handled by Hibernate.